### PR TITLE
Add Zalando (UK)

### PR DESCRIPTION
**Pull request by Nicos18**

### CODE REVIEW

**Comment by stephengroat** [comment]
improved image

oie_26182015xlojxvxi 1

**Review by stephengroat — APPROVED** [review]

File: _data/retail.yml [hunk 1]
```diff
@@ -315,6 +315,12 @@ websites:
       twitter: Wayfair
       facebook: wayfair
 
+    - name: Zalando
```
Comment (Carlgo11) [inline _data/retail.yml]: Change the name to `Zalando (UK)` as they serve multiple countries.

File: _data/retail.yml [hunk 2]
```diff
@@ -315,6 +315,12 @@ websites:
...
+      url: https://www.zalando.co.uk/
+      img: zalando.png
+      tfa: No
+      twitter: Zalando
```
Comment (Carlgo11) [inline _data/retail.yml]: Change the twitter handle to `zalando_uk`

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
Great! Thanks 👍

**Pull request merged**